ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -62,11 +65,11 @@ export class ImportCommand extends IronfishCommand {
       account.version = ACCOUNT_SCHEMA_VERSION as number
     }
 
-    const result = await client.importAccount({
-      account,
-      rescan: flags.rescan,
-    })
-
+    const rescan = flags.rescan
+    const result =
+      'spendingKey' in account && account.spendingKey
```

Comment:
Nit: Type system should be able to pick up on the ternary and the type is implied to be SpendingAccountImport on this line, and ViwAccountImport on next. Also you can change "account: account", to just "account"

Reply:
Changed `account: account` to just `account`, but still need the `as ViewAccountImport` cast on the next line